rust-client/src/main.rs

```diff
@@ -184,6 +182,14 @@ fn show(program_client: &anchor_client::Program, vault: Pubkey) -> Result<()> {
     Ok(())
 }
 
+pub fn get_current_node_clock_time(program_client: &anchor_client::Program) -> Result<u64> {
+    let rpc = program_client.rpc();
+    let a = rpc.get_account(&sysvar::clock::id())?;
```

Comment:
dont know the convention, but something other than `a` ?

Reply:
I renamed it